Idea-Archive/Sources/Utility/UIColor/HexColor.swift

```diff
@@ -0,0 +1,30 @@
+import UIKit
+
+extension UIColor {
+    convenience init(red: Int, green: Int, blue: Int, a: Int = 0xFF) {
+        self.init(
+            red: CGFloat(red) / 255.0,
+            green: CGFloat(green) / 255.0,
+            blue: CGFloat(blue) / 255.0,
+            alpha: CGFloat(a) / 255.0
+        )
+    }
+ 
+    convenience init(rgb: Int) {
+           self.init(
+               red: (rgb >> 16) & 0xFF,
+               green: (rgb >> 8) & 0xFF,
+               blue: rgb & 0xFF
+           )
+       }
```

Comment:
```suggestion
    convenience init(rgb: Int) {
       self.init(
           red: (rgb >> 16) & 0xFF,
           green: (rgb >> 8) & 0xFF,
           blue: rgb & 0xFF
       )
   }
```

복사하구 아무 처리도 안하면 탭 4칸 띄어지니 처리해주세요

Reply:
4칸 지웠어요!

Reply:
코멘트 반영하고 push한다음 해당 커밋의 url도 함께 답변해주세요

Reply:
https://github.com/Idea-Archive/Idea-Archive-iOS/pull/6/commits/91c97bd60bfbe5c2c89620a81ba234062405e608

Reply:
https://github.com/Idea-Archive/Idea-Archive-iOS/commit/91c97bd60bfbe5c2c89620a81ba234062405e608#diff-b00fecc64f647b9b4adb0b9fa886f92a94f12ab318d4b01ec6fb7d85d3139f2d

Reply:
https://github.com/Idea-Archive/Idea-Archive-iOS/pull/6/commits/91c97bd60bfbe5c2c89620a81ba234062405e608

Reply:
https://github.com/Idea-Archive/Idea-Archive-iOS/pull/6/commits/462348290fff5f80be4fdcd882220082abbf3768

해결했습니다.